examples/server_hardware.go

```diff
@@ -22,7 +23,9 @@ func main() {
 		"*")
 
 	filters := []string{""}
-	ServerList, err := ovc.GetServerHardwareList(filters, "", "", "", "")
+	ServerLists, err := ovc.GetServerHardwareList(filters, "", "", "", "")
+	total := strconv.Itoa(ServerLists.Total)
+	ServerList, err := ovc.GetServerHardwareList(filters, "", "", total, "")
```

Comment:
can you add this in library code, 
if user do not provide any start or count input then ot should get all the server list